lib/rectify/view_controller.rb

```diff
@@ -0,0 +1,40 @@
+module Rectify
+  module ViewController
+    def self.included(base_class)
+      base_class.helper_method :view
+    end
+
+    def template_view(view_object)
```

Comment:
Thinking that maybe this should be `present` rather than `template_view`. For template view objects, the call in the controller to set them up would be:

```
present Accounts::ChildrenListView.new(
  :names => %w(Amber Megan Charlie)
)
```

I was then thinking that the methods `layout_view` and `use_view` could be removed in favour of calling `present` with an extra parameter such as:

```
present :layout, ApplicationView.new(:greeting => "Hello")
present :arbitrary, MyArbitraryView.new(:something_cool => "andy")
```

That feels a little nicer in terms of the API. The template usage would stay the same:

```
p= view.random_child_link
p= view(:layout).welcome_message
p= view(:arbitrary).cool
```

We could change this to be `presenter` rather than `view` just as a way to remove confusion for Rails developers:

```
p= presenter.random_child_link
p= presenter(:layout).welcome_message
p= presenter(:arbitrary).cool
```

But then maybe we should change all references from 'View` to `Presenter` even though it is a little longer.